src/main/java/duke/command/AddToDoCommand.java

```diff
@@ -0,0 +1,30 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.task.ToDo;
+import duke.ui.Ui;
+
+/**
+ * Represents Duke's todo command
+ */
+public class AddToDoCommand extends Command {
+    public AddToDoCommand() {}
+
+    /**
+     * Adds a ToDo task to the tasklist and stores the new tasklist.
+     *
+     * @throws DukeException If user input is invalid.
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage store) throws DukeException {
+        ToDo temp = new ToDo(ui.getName());
+        tasks.add(temp);
+        store.saveToFile(tasks);
+        ui.printWithPartition("\tGot it. I've added this task:\n" + "\t  " + temp.toString()
+                + "\n\tNow you have " + Integer.toString(tasks.size()) + " tasks in the list.\n");
+
```

Comment:
Perhaps there is an unnecessary space here that could be removed? I'm not sure if the line is meant for anything as it seems to be empty line on my end
```suggestion
```